website/docs/cluster-management/managing-existing-clusters.mdx

```diff
@@ -36,146 +36,154 @@ kubectl create secret generic demo-01-kubeconfig \
 
 1. Create a new service account on the remote cluster:
 
-```yaml
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: demo-01
-  namespace: default
-```
+	```yaml
+	apiVersion: v1
+	kind: ServiceAccount
+	metadata:
+		name: demo-01
+		namespace: default
+	```
+
+1. Add RBAC permissions for the service account
+
+	<details><summary>Expand to see role manifests</summary>
```

Comment:
Do you see the same rendering issue? 

<img width="1677" alt="Screenshot 2023-02-21 at 10 32 56" src="https://user-images.githubusercontent.com/12957664/220321262-dc4f89b6-7781-4791-accb-5c3ef0c48e2e.png">

Reply:
👀 will check it out

Reply:
ah yeh forgot to leave a newline